qubes/storage/lvm.py

```diff
@@ -347,6 +387,22 @@ def _is_snapshot(self):
     def _is_volatile(self):
         return not self.snap_on_start and not self.save_on_stop
 
+    @property
+    def size(self):
```

Comment:
I'm really not sure what should be the correct solution. I see those:
 - implement setter (which in some cases will have no effect - namely after volume is already created on disk, as then actual LVM volume size is used)
 - prevent `Volume.__init__` setting `self.size` if already handled by child class (how to detect this?)